package.json

```diff
@@ -19,6 +19,14 @@
     "react-helmet": "^5.2.1"
   },
   "devDependencies": {
```

Comment:
may be it's necessary to add husky for pre-commit hook? 

like this 
```"lint-staged": {
    "*.{js,jsx}": [
      "npx eslint -c eslint.json",
      "git add"
    ]
  }
```
```"husky": {
    "hooks": {
      "pre-commit": "npx lint-staged"
    }
  }```

Reply:
Am I wrong but isn't husky just a npm package? How is it related to hooks?
Is it some type of slang I am not aware of?

Reply:
Yes)

Reply:
I install packages hasky & lint-stages, then add following code to psckage.json:
```
"lint-staged": {
    "src/**/*.{js,jsx}": [
      "eslint . --fix",
      "git add"
    ]
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
``` 
but how it should work?)

Reply:
it should be triggered hook after your commiting

Reply:
I updated commit. Can you check it?